### ISSUE TITLE
Models for 6mA methylations coming?

### ISSUE THREAD

**Issue by Kirk3gaard** [nanoporetech/megalodon#291]
Hi

Any plans for expanding to include models for 6mA methylations? 

Best regards
Rasmus

**Comment by marcus1487** [comment]
Yes. 6mA is the highest priority all-context model. We do not have a firm time table for this model as it still involves a bit of research, but will release this model once we have it ready.

**Comment by katerinaoleynikova** [comment]
any news?

**Comment by marcus1487** [comment]
Nothing at this time. This is still a very high priority, but we do not have a model released at this time.

**Comment by katerinaoleynikova** [comment]
That would be great, thx!

**Comment by smalseba** [comment]
Dear nanopore team.
Any update on this front? With LSK109 being discontinued soon, a 6mA model compatible with LSK114 (MinION, P2) becomes really important. Thx!

**Comment by znruss** [comment]
Seconded - we just did a couple runs in the new pod5 format and are seeing no first party options for actually analyzing the 6mA content of our data. It would be especially useful if a dorado version of the 6ma model is released.
